vdc/k8s-config.yaml

```diff
@@ -271,3 +271,43 @@ spec:
   resources:
     requests:
       storage: 10Mi
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: batch
+  namespace: batch-pods
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
+  replication-type: none
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: batch-storage-class-quota
+  namespace: batch-pods
+spec:
+  hard:
+    batch.storageclass.storage.k8s.io/requests.storage: 100Gi
```

Comment:
Probably not for this PR, but could we make this configurable? Given the sizes of VCF files (i.e GBs / sample), people may bump up against this before hitting more than a few hundreds of samples (1000G is ~900GB decompressed)

Reply:
Sure. Until we're confident the system is secure and correct I don't want to risk spending lots of money. This should be enough for the batch tests.

Reply:
Sounds good